navigation/navigation-compose/src/commonMain/kotlin/androidx/navigation/compose/ComposeNavigator.kt

```diff
@@ -101,3 +101,5 @@ public expect class ComposeNavigator : Navigator<Destination> {
         internal val NAME: String
     }
 }
+
+@PublishedApi internal val ComposeNavigator.Companion.name get() = ComposeNavigator.NAME
```

Comment:
This (and similar code in DialogNavigator) is because the inline funs were having issues accessing `ComposeNavigator.Companion.NAME` from js/wasmJs even though they were correctly annotated with `PublishedApi`.